openmetadata-ui/src/main/resources/ui/src/styles/tailwind.css

```diff
@@ -268,6 +268,10 @@
     @apply tw-border-b tw-border-separator;
   }
 
+  .activeCategory {
+    @apply tw-bg-primary-lite tw-border-l-2 tw-border-primary;
```

Comment:
can we use custom css/less instead of using tailwind?

Reply:
updated!